[2022-03-20 20:55:45,689] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-16T00:00:00+00:00 [queued]>
[2022-03-20 20:55:45,799] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-16T00:00:00+00:00 [queued]>
[2022-03-20 20:55:45,799] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:45,799] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2022-03-20 20:55:45,799] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:45,917] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): prepare_insert> on 2022-03-16T00:00:00+00:00
[2022-03-20 20:55:45,934] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_news', 'prepare_insert', '2022-03-16T00:00:00+00:00', '--job-id', '255', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_news.py', '--cfg-path', '/tmp/tmp5hvytm5o', '--error-file', '/tmp/tmp48yv9rti']
[2022-03-20 20:55:45,928] {standard_task_runner.py:52} INFO - Started process 495 to run task
[2022-03-20 20:55:45,937] {standard_task_runner.py:77} INFO - Job 255: Subtask prepare_insert
[2022-03-20 20:55:46,157] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_news.prepare_insert 2022-03-16T00:00:00+00:00 [running]> on host e39215f0be5b
[2022-03-20 20:55:46,379] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_news
AIRFLOW_CTX_TASK_ID=prepare_insert
AIRFLOW_CTX_EXECUTION_DATE=2022-03-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-16T00:00:00+00:00
[2022-03-20 20:55:46,391] {python.py:118} INFO - Done. Returned value was: None
[2022-03-20 20:55:46,454] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_news, task_id=prepare_insert, execution_date=20220316T000000, start_date=20220320T205545, end_date=20220320T205546
[2022-03-20 20:55:46,749] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-20 20:55:46,846] {local_task_job.py:146} INFO - Task exited with return code 0
